packages/opentelemetry-node/README.md

```diff
@@ -10,6 +10,11 @@
 [![npm dm](https://img.shields.io/npm/dm/@sentry/opentelemetry-node.svg)](https://www.npmjs.com/package/@sentry/opentelemetry-node)
 [![npm dt](https://img.shields.io/npm/dt/@sentry/opentelemetry-node.svg)](https://www.npmjs.com/package/@sentry/opentelemetry-node)
 
+This package allows you to send your NodeJS OpenTelemetry trace data to Sentry via OpenTelemetry SpanProcessors.
```

Comment:
l: We maybe also want to mention that this _only_ affects performance/transaction data, not errors, but we can also further extend on this readme later.